src/pke/lib/scheme/bgvrns/bgvrns-parametergeneration.cpp

```diff
@@ -363,19 +373,75 @@ bool ParameterGenerationBGVRNS::ParamsGenBGVRNS(std::shared_ptr<CryptoParameters
         OPENFHE_THROW(s.str());
     }
 
+    uint32_t ringDimension = cyclOrder / 2;
+    InitializeFloodingDgg(cryptoParams, numHops, ringDimension);
+
     bool dcrtBitsSet = (dcrtBits == 0) ? false : true;
```

Comment:
Question: let's say dcrtBits = 0 and dcrtBitsSet=false. dcrtBits gets a non-zero value below if (scalTech == FIXEDMANUAL), but we still use dcrtBitsSet as "false" on line 472. I understand that dcrtBits changes its value for FIXEDMANUAL, but it is very confusing. I know we are supposed to work with the changes only, but I will get rid of dcrtBitsSet to avoid any possible future issues

Reply:
Now the dcrtBitsSet logic is captured by the validator. So it is OK to remove it since the validator changes have been made.